designcompose/src/main/java/com/android/designcompose/DesignView.kt

```diff
@@ -501,6 +503,7 @@ internal fun DesignView(
                     //
                     // The interaction test document covers all of these cases.
                     interactionScope.launch {
+                        beginSection("DesignView InteractionScope")
```

Comment:
Just curious, does this work? I would have thought you would need to have the trace in onPress to get a result

Reply:
Yep :)

![image](https://github.com/google/automotive-design-compose/assets/5819722/7a193df0-7c6c-4c30-a19e-62573bcb5c29)

Blue bar at the top